bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/datamodel/FreeAtHomeDeviceChannel.java

```diff
@@ -0,0 +1,540 @@
+
+                logger.info("AQS Pressure channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1562, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            case FID_AIRQUALITYSENSOR_NO2: {
+                this.channelId = channelId;
+
+                logger.info("AQS NO2 channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1565, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            case FID_AIRQUALITYSENSOR_O3: {
+                this.channelId = channelId;
+
+                logger.info("AQS O3 channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1566, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            case FID_AIRQUALITYSENSOR_PM10: {
+                this.channelId = channelId;
+
+                logger.info("AQS PM10 channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1567, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            case FID_AIRQUALITYSENSOR_VOC: {
+                this.channelId = channelId;
+
+                logger.info("AQS VOC channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1569, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            case FID_AIRQUALITYSENSOR_PM25: {
+                this.channelId = channelId;
+
+                logger.info("AQS PM25 channel - Channel FID: {}", channelFunctionID);
+
+                FreeAtHomeDatapointGroup newDatapointGroup = new FreeAtHomeDatapointGroup();
+                newDatapointGroup.addDatapointToGroup(DATAPOINT_DIRECTION_OUTPUT, 1568, channelId, channelObject);
+                datapointGroups.add(newDatapointGroup);
+
+                break;
+            }
+            default: {
+                logger.info("Unknown channel found - Channel FID: {}", channelFunctionID);
```

Comment:
This has the information purpose for the new channel types which are detected by the binding but not implemented. It shall makes the implementation of the unknown channels easier.

This code can reoccurs, because the detection of the channels happens separately for each channel.

Reply:
If I understand you, this would mean that unimplemented channel will result in messages like this reappearing over and over again in the logs. This should not happen, so I would propose to change log level to **debug**.